sample/src/main/java/com/springboot/hello/sample/controller/Controller.java

```diff
@@ -0,0 +1,23 @@
+package com.springboot.hello.sample.controller;
+
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+public class Controller {
+	
+	@RequestMapping("/")
```

Comment:
Always better to have a context path like `/hello` rather than just having a resource at  the root level (`/`)

Reply:
BTW why do you need this?